### ISSUE TITLE
Getting error using OC USB installer

### ISSUE THREAD

**Issue by onoakino** [chris1111/HP-Probook-EliteBook-Package-Creator-OC#162]
Device: HP Elitebook 2570p.
Issue: USB OC Installer crashes.

Created a USB opencore installer using the lates package creator OC. The version of MacOS is 12.6.3. Here's the log displayed

[https://imgur.com/BXTxRtA](url)

![usbOCerror](https://user-images.githubusercontent.com/24814387/219932399-1b62618b-c001-4c85-88e1-850c5542f294.jpg)

**Comment by chris1111** [comment]
Sorry but I dont understanding the issue

**Comment by onoakino** [comment]
Sorry if I was not very clear. The installer stops at that boot screen, and then would reboot. 

The installer I made is for the OC installer, not the Clover installer. The MacOS version I used in creating the OC installer is 12.6.3.

**Comment by chris1111** [comment]
what is the your wifi?

**Comment by onoakino** [comment]
BCM94352HMB/AW-CE123H

**Comment by chris1111** [comment]
picture looks from booting usb installer

**Comment by onoakino** [comment]
Yes. When I boot from the USB installer, that's the error I get.

**Comment by chris1111** [comment]
This is normal, let it boot and at the end you will be in the install macOS
